src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DoNotUseEnumerableMethodsOnIndexableCollectionsInsteadUseTheCollectionDirectly.Fixer.cs

```diff
@@ -80,6 +86,27 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
                                     diagnostic);
         }
 
+        private static bool TryExtractInvocationOperation(IOperation originalOperation, [NotNullWhen(returnValue: true)] out IInvocationOperation? invocation)
```

Comment:
This is for extracting invocation operation due to diagnostic location change for conditional access expressions

Reply:
Please add a comment in code.